rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -1192,7 +1192,7 @@ void ppu_thread::dump_regs(std::string& ret) const
 		}
 	}
 
-	fmt::append(ret, "CIA: 0x%08llx\n", cia);
+	fmt::append(ret, "CIA: 0x%08x\n", cia);
```

Comment:
0x%07x suffices. It's rarely above 256mb.

Reply:
Won't it be ugly this way?

Reply:
The always added 0 in logs looked odd there IMO. It's also how the debugger prints PPU instruction addresses now in which the added 0 looked odd there as well.

Reply:
Removed 0s prefix